test/unit/widgets/peoplePicker.test.ts

```diff
@@ -53,7 +143,53 @@ describe('PeoplePicker.render', () => {
       }
     `)
   })
+
+  it('.. type index ...', () => {
+    const typeIndex = 'publicTypeIndex'
+    const groupPickedCb = () => {}
+    const options = { selectedGroup: {} }
+    const element = document.createElement('p')
+    const peoplePicker = new PeoplePicker(
+      element,
+      typeIndex,
+      groupPickedCb,
+      options
+    )
+    jest.clearAllMocks()
+    const spyOnNowOrWhenFetched = jest
+      .spyOn(fetcher, 'nowOrWhenFetched')
+      .mockImplementationOnce(() => {
+        return Promise.resolve('book')
+      })
+
+    peoplePicker.render()
+    // expect(spyOnNowOrWhenFetched).toBeCalled()
+    // expect(spyOnNowOrWhenFetched).toReturnWith('book')
+    expect(spyOnNowOrWhenFetched).toBeCalledTimes(1)
+    // expect(spyAny).toBeCalled() // not getting called.
+    // expect(spySecondAny).toBeCalled()
+    // expect(spyLoad).toBeCalled()
+
+    expect(peoplePicker.render()).toMatchInlineSnapshot(`
```

Comment:
In general I think we should try to avoid inline snapshots, and have actual snap files using `toMatchSnapshot`.

Reply:
I will change this.